requirements/requirements.txt

```diff
@@ -1,8 +1,8 @@
 mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
-inflection~=0.5
+inflection~=0.3
```

Comment:
im following up with a PR that removes this dep

Reply:
It did not really show up as conflict. Within our image I am manually installing all python packages as during the build step we do not have pip.

inflection is still being installed by default which is the reason pip check reports it as possible conflict. If the requirement is removed, I will remove it from the default install in our image.

Basically only the pyxdg package really was a conflict and also errored out within the logs

Reply:
Actually, I was the one asking if it could be removed as it is kind of incompatible with the buildroot system. Will delete the package.